agent/ansible/pbench/agent/galaxy.yml

```diff
@@ -9,7 +9,7 @@ namespace: pbench
 name: agent
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 1.0.3
+version: 1.0.4
```

Comment:
@ndokos, is there no versioning done for the server side?

Reply:
I should open an issue on that: the agent roles are on galaxy, but the server roles are not - yet. They need renaming and reorganization - and documentation - to satisfy galaxy requirements. So no versioning yet - but it's coming.

Reply:
Tracked in #2294.